joomla/joomla-cms
# remove insert module button

Pull Request for Issue 10525 .
#### Summary of Changes

This PR removes the ability to insert a module into 
Contact Form information
Tag description
Category description
Banner Description
#### Testing Instructions

Open above forms and see that the module button is not present in the tinymce toolbar
